WordPress/Classes/ViewRelated/System/NoticePresenter.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import UIKit
+import WordPressFlux
+
+class NoticePresenter: NSObject {
+    private let store: NoticeStore
+    private var storeReceipt: Receipt?
+
+    private var isPresenting = false
```

Comment:
We'd have to test it, but I believe with the new NoticeStore design, the `isPresenting` state is redundant. Since the NoticeStoreStateState only contains the notice that should be presented, the presenter should ensure that's always the presented one.

Reply:
I think you're right – removing it now, much cleaner :)